src/libs/ReportUtils.js

```diff
@@ -1711,7 +1719,7 @@ function canLeaveRoom(report, isPolicyMember) {
  * @returns {string}
  */
 function getWhisperDisplayNames(participants) {
-    const isWhisperOnlyVisibleToCurrentUSer = this.isCurrentUserTheOnlyParticipant(participants);
+    const isWhisperOnlyVisibleToCurrentUSer = isCurrentUserTheOnlyParticipant(participants);
```

Comment:
🤯 can you explain to me how this is different 🙏 

I see that the function uses som variable `sessionEmail`, what happens with that variable when you call this function without `this`?

Reply:
Lol, nevermind, I just noticed this is JS and not PHP

Reply:
On dev this refers to the module (ReportUtils), but when it's built, minified version, it's not the case so it crashes.

Reply:
You can follow the convo here hehe https://expensify.slack.com/archives/C049HHMV9SM/p1682089468352009

Reply:
Gracias!